openmls/src/framing/message.rs

```diff
@@ -1,127 +1,212 @@
+use std::io::BufReader;
```

Comment:
Can you add here, or in the `framing` module, a description of the different types and the intended workflow? We need a) the one for the public API, and b) one for the internal types.